packages/components/e2e/component-objects/src/main/java/org/talend/component/Component.java

```diff
@@ -37,20 +43,22 @@ public Component(WebDriver driver, String name, String selector) {
         this.jsExec = (JavascriptExecutor) driver;
         this.name = name;
         this.selector = selector;
+        this.wait = new WebDriverWait(this.driver, 1);
```

Comment:
No wait method should be used. 
It prevents reproducible builds and introduces latency.